src/jit/MemoryInl.h

```diff
@@ -515,8 +533,24 @@ static void emitLoadLaneSIMD(sljit_compiler* compiler, Instruction* instr)
         return;
     }
 
+#if (defined SLJIT_64BIT_ARCHITECTURE && SLJIT_64BIT_ARCHITECTURE)
     sljit_emit_simd_lane_mov(compiler, SLJIT_SIMD_LOAD | simdType, dstReg, loadOperation->index(), addr.memArg.arg, addr.memArg.argw);
+#else /* !SLJIT_64BIT_ARCHITECTURE */
+    if (simdType == (SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_64)) {
+        sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_R1, 0, addr.memArg.arg, addr.memArg.argw);
```

Comment:
On ARM, a 64 bit register can be loaded directly, so this should be just one instruction. If lane is 0, it is dstReg, otherwise getHighRegister(dstReg)